setup.cfg

```
@@ -26,6 +26,7 @@ classifiers =
     Topic :: Utilities
 keywords = test, unittest
 project_urls =
+    Changelog=https://docs.pytest.org/en/stable/changelog.html
```

there's a full list of all the special icons warehouse supports:

https://github.com/pypa/warehouse/blob/851e4c37bc1a9d1ef299e9300c7aa3ffb7cb09d5/warehouse/templates/packaging/detail.html#L20-L58

we may want to add some more eg https://twitter.com/pytestdotorg

---

Should it be then: `twitter=https://twitter.com/pytestdotorg`?

Could you please add an inline suggestion if you are sure about the syntax @graingert ?

---

```suggestion
    Changelog=https://docs.pytest.org/en/stable/changelog.html
    Twitter=https://twitter.com/pytestdotorg
```

@nicoddemus do we want the coverage on here? or anything else from that list?

---

IMHO this is good enough... no need to add coverage/CI information here, I think.